remoteappmanager/static/js/filters.js

```diff
@@ -0,0 +1,22 @@
+define([
```

Comment:
we need a standard place for vue stuff at this point. I have the same problems with components.
I would say create a directory /static/js/vue/. Inside, put filters.js. I will also do the same, and have a subdir components, where I'll put the initial toolkit.